_topic_maps/_topic_map.yml

```
@@ -2098,12 +2098,19 @@ Topics:
     File: nodes-nodes-managing-max-pods
   - Name: Using the Node Tuning Operator
     File: nodes-node-tuning-operator
-  - Name: Remediating nodes with the Self Node Remediation Operator
-    File: eco-self-node-remediation-operator
-  - Name: Deploying node health checks by using the Node Health Check Operator
-    File: eco-node-health-check-operator
-  - Name: Using the Node Maintenance Operator to place nodes in maintenance mode
-    File: eco-node-maintenance-operator
+  - Name: Remediation, Fencing, and Maintentance
```

Checking the capitalization here, should it be as follows? Also small typo in "maintenance" :)
```suggestion
  - Name: Remediation, fencing, and maintenance
```